### PR TITLE
##Feature mobile menu

### PR DESCRIPTION
**Pull request by sarazaToussaint**
In this project we:

> Add Eslint in our repo
> Add index.js file to make our hamburger button work once a user clicks on it
> Add codes in makeup and stylesheet files for the mobile menu

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  document.querySelector('nav').style.display = 'none';
+  document.querySelector('#hamburger').style = 'display: block';
+  document.querySelector('body').style = 'overflow: scroll';
+};
+
+closeItems.onclick = function closeItem() {
+  document.querySelector('nav').style.display = 'none';
+  document.querySelector('#hamburger').style = 'display: block';
+  document.querySelector('body').style = 'overflow: scroll';
+};
```
Comment (ShadyShawkat) [inline index.js:21]: - [ ] [OPTIONAL] Please make sure to use `ES6` arrow functions instead of the `function` keyword, which is considered good practice.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @sarazaToussaint, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [ ] Kindly make sure to add the `MIT` License and the link to it in your `README.md`, you can see it [here](https://github.com/microverseinc/readme-template).
- [ ] _Check the comments under the review for more changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sarazaToussaint** [comment]
I made the required changes except changing the functions to the arrows function which is optional. 
Thank you for the review

**Review by Mre55 — APPROVED** [review]
**Hi @sarazaToussaint,**

# Status: APPROVED 🎉  💪 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

Alt Text

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**